<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryEditorWidgetClass</class>
 <widget class="QWidget" name="QueryEditorWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>436</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="dbChooser"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="pathLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="runButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>R&amp;un</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/img/player_play.png</normaloff>:/img/player_play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="shortcut">
      <string>F5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QueryTextEdit" name="editor">
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
      <property name="html" stdset="0">
       <string/>
      </property>
     </widget>
     <widget class="ResultView" name="tabView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionEnqueue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enqueue</string>
   </property>
   <property name="toolTip">
    <string>Puts the query in a threaded queue</string>
   </property>
  </action>
  <action name="actionLogQuery">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log query</string>
   </property>
  </action>
  <action name="actionClearOnSuccess">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clear if success</string>
   </property>
   <property name="toolTip">
    <string>Clears the query if it is runned with success</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QueryTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>../src/widgets/querytextedit.h</header>
  </customwidget>
  <customwidget>
   <class>ResultView</class>
   <extends>QWidget</extends>
   <header>../src/widgets/resultview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
